frontend/ui/widget/container/inputcontainer.lua

```diff
@@ -269,6 +269,64 @@ function InputContainer:onGesture(ev)
     end
 end
 
+-- Will be overloaded by the Gestures plugin, if enabled, for use in _onGestureFiltered
+function InputContainer:isGestureAlwaysActive(ges, multiswipe_directions)
+    -- If the plugin isn't enabled, block everything, we'll restore standard behavior on resume.
+    return false
+end
+
+-- Filtered variant that only lets is_always_active touch zones through.
+-- (This is used by the "toggle_touch_input" Dispatcher action).
+function InputContainer:_onGestureFiltered(ev)
+    for _, tzone in ipairs(self._ordered_touch_zones) do
+        if self:isGestureAlwaysActive(tzone.def.id, ev.multiswipe_directions) and tzone.gs_range:match(ev) and tzone.handler(ev) then
+            return true
+        end
+    end
+    -- No ges_events at all, although if the need ever arises, we could also support an is_always_active field for those ;).
+    if self.stop_events_propagation then
+        return true
+    end
+end
+
+-- NOTE: This works because barely anything overloads onGesture.
+--       We would need to be slightly more creative if things did
+--       (i.e., don't stop the event and swap self.onGesture in every instance
+--       while still only swapping Input.onGesture once...)
+function InputContainer:onIgnoreTouchInput(toggle)
```

Comment:
Might need a bit more context before your NOTE: :) at least saying what its "work" is :) and that it should be triggered by the "toggle_touch_input" gesture only.